docs/metal_gateway_create.md

```diff
@@ -23,11 +23,11 @@ metal gateway create -p <project_UUID> --virtual-network <virtual_network_UUID>
 ### Options
 
 ```
-  -h, --help                       help for create
-  -r, --ip-reservation-id string   UUID of the Public or VRF IP Reservation to assign.
-  -s, --private-subnet-size int    Size of the private subnet to request (8 for /29)
-  -p, --project-id string          The project's UUID. This flag is required, unless specified in the config created by metal init or set as METAL_PROJECT_ID environment variable.
-  -v, --virtual-network string     UUID of the Virtual Network to assign.
+  -h, --help                        help for create
+  -r, --ip-reservation-id string    UUID of the Public or VRF IP Reservation to assign.
+  -s, --private-subnet-size int32   Size of the private subnet to request (8 for /29)
```

Comment:
Yes reverted.

Reply:
It looks like the code changes were reverted, but not the help text; this help text is still showing in the PR view.  The code changes should remain (it's good to define the var as `Int32VarP` instead of `IntVarP`).  The only thing that should be reverted is the help text.

Reply:
I thought the doc would automatically be updated if we change the variable names. Would make the change

Reply:
Done

Reply:
@aayushrangwala I'm sorry, you were right and I was wrong; I forgot that the docs are generated and that the documentation change to `int32` is coming from cobra and not from a manual change.  Could you reset your branch to [937c467](https://github.com/equinix/metal-cli/pull/376/commits/937c4670b7b714d9a4b04a30201174640b3c111c)?  That commit passed all CI and is good to merge.

Reply:
Done, thanks